ash/src/lib.rs

```diff
@@ -25,6 +25,12 @@
 //! # Ok(()) }
 //! ```
 //!
+//! ## Getting started
+//! Load the Vulkan library at the default location using [`Entry::new()`][EntryCustom<_>::new()],
+//! or at a custom location using [`Entry::with_library("path/to/vulkan")`][EntryCustom<_>::with_library()].
```

Comment:
Rustdoc doesn't seem capable of finding functions on type aliases yet :/

```rust
29 | //! Load the Vulkan library at the default location using [`Entry::new()`],
   |                                                            ^^^^^^^^^^^^^^ the type alias `Entry` has no function named `new`
```
So we're forced to specify `EntryCustom<_>::` directly.